nextstrain/cli/runner/aws_batch/__init__.py

```diff
@@ -83,6 +104,8 @@ def run(opts, argv, working_volume = None) -> int:
             name       = run_id,
             queue      = opts.job_queue,
             definition = opts.job_definition,
+            cpus = int(opts.cpus),
+            memory = int(opts.memory),
```

Comment:
Adding `type = int` to the `add_argument` calls will ensure the types of `opts.cpus` and `opts.memory` are always correct and remove the need for the `int(…)` calls here. It will also cause argparse to throw an error early if someone provides a non-integer value.

Reply:
This is much better.